shapely/ops.py

```diff
@@ -718,3 +723,50 @@ def orient(geom, sign=1.0):
 
     """
     return shapely.orient_polygons(geom, exterior_cw=sign < 0)
+
+
+def get_segments(geometry):
+    """Return the constituent segments a line.
```

Comment:
```suggestion
    """Get segments of each linear geometry object.
```

Looking at consistency of docstrings of `get_parts` and `get_coordinates`